docs/markdown_files/relecov_tools.md

```diff
@@ -1,186 +1,90 @@
 # relecov-tools
 
+[![python_lint](https://github.com/BU-ISCIII/relecov-tools/actions/workflows/python_lint.yml/badge.svg)](https://github.com/BU-ISCIII/relecov-tools/actions/workflows/python_lint.yml)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+> THIS REPO IS UNDER ACTIVE DEVELOPMENT.
```

Comment:
Replace this line for something like `"Find more information about the tools here: https://github.com/BU-ISCIII/relecov-tools"`